packages/mobile/src/screens/form-screen/FormScreen.tsx

```diff
@@ -64,6 +65,7 @@ export const FormScreen = (props: FormScreenProps) => {
         backgroundColor='white'
         borderTop='default'
         gap='m'
+        style={css({ position: 'absolute', bottom: 0, left: 0, right: 0 })}
```

Comment:
looks ok to me, but we should check this doesn't break other screens

Reply:
oo good call - it breaks things! with this absolute position, other usages of FormScreen have the same problem i was running into where the footer overlaps some of the options.
height: 70% fixes this on the ChatBlastSelectContentScreen so will leave it at that for now.